lib/vm/src/trap/traphandlers.rs

```diff
@@ -399,8 +417,9 @@ pub unsafe fn raise_lib_trap(trap: Trap) -> ! {
 ///
 /// # Safety
 ///
-/// Only safe to call when wasm code is on the stack, aka `wasmer_call` or
-/// `wasmer_call_trampoline` must have been previously called.
+/// Only safe to call when wasm code is on the stack, aka `catch_traps` must
+/// have been previously called. Additionally no Rust destructors can be on the
```

Comment:
"been previously called" --> "been previously called and not returned" here too, and in a few other comments too.